config/miden-node.toml

```diff
@@ -17,6 +17,6 @@ store_url = "http://localhost:28943"
 [store]
 # port defined as: sum(ord(c)**p for (p, c) in enumerate('miden-store', 1)) % 2**16
 endpoint = { host = "localhost", port = 28943 }
-database_filepath = "miden-store.sqlite3"
-genesis_filepath = "genesis.dat"
-blockstore_dir = "blocks"
+database_filepath = "/opt/miden/miden-store.sqlite3"
+genesis_filepath = "/opt/miden/genesis.dat"
+blockstore_dir = "/opt/miden/blocks"
```

Comment:
This is now assuming that we are on Linux-based systems, right? (i.e., this won't work on Windows). This is fine as we are currently not targeting Windows anyways, but we should probably mention this more explicitly in the docs.

Reply:
What do you have in mind? The docs mention this is an _example_:

```
To run the node you will need to provide a configuration file. We have an example config file in [node/miden-node.toml](https://github.com/0xPolygonMiden/miden-node/blob/main/node/miden-node.toml). Then, to run the node, run:
```

Not sure what else is necessary there.